plugins/network/upnpc_

```diff
@@ -20,7 +20,11 @@ Linux systems with upnpc installed (miniupnpc package).
 
 =head1 CONFIGURATION
 
-None needed.
+If you do not want to show the link maximum bitrates, add the following
+plugin-configuration:
+
+  [upnpc*]
```

Comment:
Maybe add the final underscore in order to keep it focused?

Reply:
No, as the plugin also supports `upnpc` as a shorthand for `upnpc_multi`

Reply:
Good point!